tubearchivist/umbrel-app.yml

```
@@ -0,0 +1,31 @@
+manifestVersion: 1
+id: tubearchivist
+category: media
+name: Tube Archivist
+version: "0.5.7"
+tagline: Self hosted YouTube media server
+description: >-
+  Once your YouTube video collection grows, it becomes hard to search and find a specific video. 
+  That's where Tube Archivist comes in: By indexing your video collection with metadata from YouTube, you can organize, search and enjoy your archived YouTube videos without hassle offline through a convenient web interface. 
+  
+
+  Core functionality includes:
+    - Subscribe to your favorite YouTube channels
+    - Download Videos using yt-dlp
+    - Index and make videos searchable
+    - Play videos
+    - Keep track of viewed and unviewed videos
+developer: tubearchivist
+website: https://www.tubearchivist.com/
+dependencies: []
+repo: https://github.com/tubearchivist/tubearchivist
+support: https://github.com/tubearchivist/tubearchivist/issues
+port: 8079
+gallery: []
+path: ""
+defaultUsername: "admin"
+defaultPassword: "umbrel"
```

Use of generated password.
```suggestion
deterministicPassword: true
```